ChangeLog.md

```diff
@@ -3,6 +3,25 @@
 
 ## Version 10.11.0
 
+### New features
+1. Added support for include and exclude regex flags, which allow pattern matching on the entire paths.
+2. Added dry run mode for copy, remove, and sync. This feature allows the user to visualize the changes before committing them.
+3. For SMB aware locations, preserve-smb-info flag is now true by default.
+4. Improved how folder lmts are obtained to allow time-based filters for folders.
```

Comment:
I _want_ to call this a bugfix, but I _want_ to call this a feature. a featugfix.

Reply:
yah.. it can spin both ways. It's new capability, so let's call it a feature